app/main/handle-external-link.ts

```diff
@@ -125,16 +131,30 @@ export default function handleExternalLink(
       url: url.href,
       downloadPath,
       async completed(filePath: string, fileName: string) {
+        const notificationTitle = t.__("Download Complete");
```

Comment:
We should use our standard capitalization pattern.

```suggestion
        const notificationTitle = t.__("Download complete");
```

Reply:
Fixed!